externs/es3.js

```diff
@@ -513,6 +513,7 @@ Array.prototype.push = function(var_args) {};
  * Transposes the elements of an array in place: the first array element becomes the
  * last and the last becomes the first.
  *
+ * @return {!Array<T>}
  * @this {{length: number}}
  * @modifies {this}
  * @see http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/reverse
```

Comment:
While we are here, can you fix this URL to be http://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Array/slice ?